src/points/point-manager.ts

```diff
@@ -1,63 +1,116 @@
 import { LogLevel } from "@sentio/sdk";
 import { EthContext } from "@sentio/sdk/eth";
-import { MISC_CONSTS, PENDLE_POOL_ADDRESSES } from "../consts.js";
-import { EVENT_POINT_INCREASE, POINT_SOURCE, POINT_SOURCE_YT } from "../types.js";
+import {
+  MISC_CONSTS,
+  PENDLE_POOL_ADDRESSES,
+  MULTIPLIERS,
+  V1_END_TIMESTAMP,
+} from "../consts.js";
+import {
+  EVENT_POINT_INCREASE,
+  POINT_SOURCE,
+  POINT_SOURCE_SY,
+  POINT_SOURCE_YT,
+} from "../types.js";
 
 /**
  *
  * @param amountEEthHolding amount of eEth user holds during the period
- * @param holdingPeriod amount of time user holds the eEth
+ * @param holdingStartTimestamp start timestamp of the holding period
+ * @param holdingEndTimestamp end timestamp of the holding period
  * @returns Zircuit point
  *
  * @dev to be reviewed by Zircuit team
  */
 function calcPointsFromHolding(
   amountEEthHolding: bigint,
   holdingStartTimestamp: bigint,
-  holdingEndTimestamp: bigint,
+  holdingEndTimestamp: bigint
 ): bigint {
-  const campaignStartTime = 1713373200n // 4/17 13:00 EST
-  const campaignEndTime = 1714582800n // 5/1 13:00 EST
-  const campaignMultiplier = 3n
-  const baseMultiplier = 2n
+  const campaignStartTime = MULTIPLIERS.campaign.startTimestamp;
+  const campaignEndTime = MULTIPLIERS.campaign.endTimestamp;
+  const campaignMultiplier = MULTIPLIERS.campaign.multiplier;
+  const baseMultiplier = MULTIPLIERS.multiplier;
+  const baseFactor = MULTIPLIERS.baseFactor;
 
```

Comment:
```suggestion
  const cuttoffTimestamp =  1724284800n // 22/07 12:00 AM GMT

  if(holdingStartTimestamp >= cuttoffTimestamp) return BigInt(0)
  if(holdingEndTimestamp >= cuttoffTimestamp) holdingEndTimestamp = cuttoffTimestamp
```

Reply:
@Junochiu just proposing here so we avoid redeploying the processors later on, we should handle the expiry date from now. also, the cuttoffTimestamp can be defined in the constant file instead.